content/kubermatic/main/tutorials-howtos/cluster-rbac/_index.en.md

```diff
@@ -0,0 +1,108 @@
+and simplify access control across the user cluster. All of the default roles and cluster roles are labeled
+with `component=userClusterRole`.
+
+**Permission on cluster level:**
+
+| Default ClusterRole | Description                                                                                                                                                                                                                          |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cluster-admin       | Allows admin access. Allows read/write access to most resources in a namespace, including creating roles and role bindings within the namespace. This role does not allow write access to resource quota or to the namespace itself. |
+| edit                | Allows read/write access to most objects in a namespace. This role does not allow viewing or modifying roles or role bindings. However, this role allows accessing secrets and running pods as any service account in the namespace  |
+| view                | Allows read-only access to see most objects in a namespace. It does not allow viewing roles or role bindings.                                                                                                                        |
+| list-namespaces     | Allows to list namespaces                                                                                                                                                                                                            |
+
+**Permission on namespace level:**
+
+| Default Role     | Description                                                                                                                                         |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| namespace-admin  | Allows admin access. Allows read/write access to most resources in a namespace.                                                                     |
+| namespace-editor | Allows read/write access to most objects in a namespace. This role allows accessing secrets and running pods as any service account in the namespace|
+| namespace-viewer | Allows read-only access to see most objects in a namespace.                                                                                         |
+
+
+
+# Manage User's permissions
+You can grant permission to a group by clicking on `add Bindings`.
+![Grant permission to a user](/img/kubermatic/main/ui/rbac_user_binding.png?classes=shadow,border "Grant permission to a user")
+
+{{% notice info %}}
+The cluster owner is automatically connected to the `admin` cluster role.
```

Comment:
Is this true? Isn't the owner assigned `cluster-admin`?

Reply:
good catch !